week2_PS_draft.ipynb

```diff
@@ -0,0 +1,523 @@
+{
```

Comment:
<p>change line 7 to: avg_monthly_precip = np.loadtxt(fname = "data/avg-monthly-precip.txt")</p><p> </p><p>change line 12 to: months = np.genfromtxt("data/months.txt", dtype='str')</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-MzCorona/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>